src/cryptoadvance/specter/server_endpoints/nodes.py

```diff
@@ -0,0 +1,333 @@
+            else:
+                node.rename(node_name)
+        elif action == "forget":
+            if not node_alias:
+                flash("Failed to deleted node. Node isn't saved", "error")
+            elif len(app.specter.node_manager.nodes) > 1:
+                app.specter.node_manager.delete_node(node, app.specter)
+                if bool(request.form.get("remove_datadir", False)):
+                    shutil.rmtree(os.path.expanduser(node.datadir), ignore_errors=True)
+                flash("Node deleted successfully")
+                return redirect(
+                    url_for(
+                        "nodes_endpoint.node_settings",
+                        node_alias=app.specter.node.alias,
+                    )
+                )
+            else:
+                flash(
+                    "Failed to deleted node. Specter must have at least one node configured",
+                    "error",
+                )
+        elif action == "stopbitcoind":
+            try:
+                node.stop()
+                time.sleep(5)
+                flash("Specter stopped Bitcoin Core successfully")
+            except Exception:
+                try:
+                    flash("Stopping Bitcoin Core, this might take a few moments.")
+                    node.rpc.stop()
+                except Exception as e:
+                    flash(f"Failed to stop Bitcoin Core {e}", "error")
+        elif action == "startbitcoind":
+            if node.start(timeout=120):
+                flash("Specter has started Bitcoin Core")
+            else:
+                flash("Specter failed to start the node...", "error")
+        elif action == "uninstall_bitcoind":
+            try:
+                node.stop()
+                shutil.rmtree(
+                    os.path.join(app.specter.data_folder, "bitcoin-binaries"),
+                    ignore_errors=True,
+                )
+                if bool(request.form.get("remove_datadir", False)):
+                    shutil.rmtree(os.path.expanduser(node.datadir), ignore_errors=True)
+                flash(f"Bitcoin Core uninstalled successfully")
+                app.specter.node_manager.delete_node(node, app.specter)
+                return redirect(
+                    url_for(
+                        "nodes_endpoint.node_settings",
+                        node_alias=app.specter.node.alias,
+                    )
+                )
+            except Exception as e:
+                flash(f"Failed to remove Bitcoin Core, error: {e}", "error")
+
+    return render_template(
+        "node/internal_node_settings.jinja",
+        node=node,
+        node_alias=node_alias,
+        specter=app.specter,
+        rand=rand,
+    )
+
+
+@nodes_endpoint.route("/internal_node_logs/<node_alias>/", methods=["GET"])
+@login_required
+def internal_node_logs(node_alias):
+    node = app.specter.node_manager.get_by_alias(node_alias)
+    logfile_location = os.path.join(node.datadir, "debug.log")
```

Comment:
Ahh, i guess this needs a change as it points to the old location.

Reply:
Not sure what you mean here? This now takes the node datadir argument, so it's the right location

Reply:
Hmmm, it didn't worked for me and i haven't checked this line good enough and simply assumed it's wrong. Doesn't look like this. Looks good, my fault!